### PR TITLE
chore: dependabot wdio update throwing unexpected e2e error 

### PR DESCRIPTION
**Pull request by castastrophe**
## dependabot wdio update patch

After the `@wdio/spec-reporter` bump to 7.6.1 (#1648), `npm run e2e` was throwing:
`[0-0] 2021-06-02T15:24:19.381Z ERROR @wdio/local-runner: Failed launching test session: TypeError: Cannot read property '7372259ede3e4b1ac864f644191b3ccbf393a154' of undefined`

### What has changed and why

- Bumped versions for local-runner and spec-reporter to be in-sync
- Fix bad if logic for the IE browser checks in e2e.js files
- Switch modal to use a promise instead of waits
- Add the missing polyfill for pfe-modal that was failing it's IE11 run
- Bump the ratio on pfe-clipboard (failing on typography diffs)

### Testing instructions

- [x] *npm run e2e pfe-accordion*
- [x] *npm run e2e pfe-modal*

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Tests have been updated to cover these changes.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-modal/src/pfe-modal.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Import polyfills: filter
+import "./polyfills--pfe-modal.js";
```
Comment (kylebuch8) [inline elements/pfe-modal/src/pfe-modal.js:2]: @castastrophe - Should this be pulled out into a separate PR? Was it causing the e2e to fail?

Comment (castastrophe) [reply elements/pfe-modal/src/pfe-modal.js:2]: It was causing e2e to fail in IE11 for pfe-modal. We can still pull it out though if that's easier.

Comment (kylebuch8) [reply elements/pfe-modal/src/pfe-modal.js:2]: Let's just leave it in.

**Review by kylebuch8 — APPROVED** [review]
Leaking Gutters Torrential Mess
